File: Python/Week 1/activity1.py
#!/usr/bin/python

def findMinMax(temperatures):
    max_value = temperatures[0]
    min_value = temperatures[0]

    for i in range(len(temperatures)):
        if temperatures[i] > max_value:
            max_value = temperatures[i]
        if temperatures[i] < min_value:
            min_value = temperatures[i]

    return max_value, min_value


def main():
    temperatures = [22, 30, 25, 28, 35, 31, 27]
    values = findMinMax(temperatures)
    print("Max Temperature: {}°C, Min Temperature: {}°C".format(
        values[0], values[1]))


if __name__ == '__main__':
    main()

File: Java/Week 1/activity_1.java
class activity_1
{
    public static void main(String[] args)
    {
        int[] temps = {22, 30, 25, 28, 35, 31, 27};
        int numEl = 7;
        int j;

        int maxValue = temps[0];
        int minValue = temps[0];
        

        for(j=0; j<numEl; j++)
            if (temps[j] > maxValue)
                maxValue = temps[j];

            else if (temps[j] < minValue)
                minValue = temps[j];    

        System.out.println("Max Temperature: " + maxValue + "\u00B0C, Min Temperature: " + minValue + "\u00B0C\n");

    }
    

}